addons/Navigation_Bar/src/presenter.js

```diff
@@ -578,9 +578,12 @@ function AddonNavigation_Bar_create() {
         }
 
         var currentElementStyle = isCurrentElement ? "navigationbar-element-current" : "navigationbar-element";
+        var currentElementId = isCurrentElement ? 'current-page-number' : '';
+        var addonID = presenter.configuration.ID;
+        var elementId = currentElementId ? ' id="' + currentElementId + (addonID ? '-' + addonID : '') + '"' : '';
```

Comment:
`
var elementId = '';
if (isCurrentElement) {
var addonID = presenter.configuration.ID;
elementId = "id="current-page-number" + (addonID ? '-' + addonID : '') + ' " ';
}


`

Reply:
Wydaje mi się, że czytelność można poprawić - zagnieżdzone conditional operatory przy znakach " i ' trochę mi się mieszają. Ale, nie będę się upierał :)
Chociaż można by się przynajmniej pozbyć var currentElementId i w linijce 583 sprawdzać po prostu isCurrentElement (skoro currentElementId jest puste to nigdzie później nie używamy i tak tego pustego stringa zapisane w linijce 581).

Reply:
Poprawione według sugestii